test/e2e/specs/authentication/authentication__sms.ts

```diff
@@ -1,8 +1,7 @@
 /**
- * @group calypso-release
+ * @group authentication
  *
- * One possible problem with this spec is that SMS OTP can only be requested once per minute.
- * In addition, anecdotally it appears that any login attempt during this one-minute blackout period resets the timer, though this isn't confirmed.
+ * One real problem with this spec is that SMS OTP can only be requested once every 5 minutes as per the backend code.
  * What this means is that in rare cases where Pre-Release Tests are lined up one after another in the queue, it may result in an unexpected failure of this spec, which would appear to be a flaky test failure to the developer.
  *
  * It may be necessary to keep a close eye on this test and immediately pull the test from rotation if we find the flakiness exceeds an acceptable level.
```

Comment:
```
* What this means is that in rare cases where Pre-Release Tests are lined up one after another in the queue, it may result in an unexpected failure of this spec, which would appear to be a flaky test failure to the developer.
 *
 * It may be necessary to keep a close eye on this test and immediately pull the test from rotation if we find the flakiness exceeds an acceptable level.
```
Not a blocking comment, but now that we're doing a separate build, do these comments still make sense?

Reply:
Thanks, I reworte the sentence above and didn't re-check whether rest of the sentences made sense 🤦